packages/core/src/internal/clients/pubsub/AbstractPubsubClient.ts

```diff
@@ -149,6 +151,7 @@ export abstract class AbstractPubsubClient<TGrpcError>
       topicName: topicName,
       onItem: onItem,
       onError: onError,
+      onDiscontinuity: options.onDiscontinuity,
```

Comment:
about 10 lines up from here is where we create the no-op implementations of these if the caller didn't pass one in, so that from this point forward they don't have to be nullable.